go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1394,6 +1390,39 @@ func newSharedGateway(nodeName string, subnets []*net.IPNet, gwNextHops []net.IP
 			// no service OpenFlows, request to sync flows now.
 			gw.openflowManager.requestFlowSync()
 		}
+
+		// Add a neighbour entry on the K8s node to map dummy IPs with MACs. This is
+		// required because the dummy next hop does not exist on the network and will not
+		// respond to an ARP, so to route we need an entry. Additionally, the OVN dummy IP
+		// is not assigned to its interface, so we also need a fake entry for that
+		link, err := util.LinkSetUp(gwBridge.bridgeName)
+		if err != nil {
+			return fmt.Errorf("unable to get link for %s, error: %v", gwIntf, err)
+		}
+
+		var neighborIPs []string
+		if config.IPv4Mode {
+			neighborIPs = append(neighborIPs, types.V4OVNMasqueradeIP, types.V4DummyNextHopMasqueradeIP)
```

Comment:
why `V4OVNMasqueradeIP` is added?

Reply:
Doh, I gues I confused the host masquerade with the OVN masquerade IP. I still think that replacing `the OVN dummy IP` with `the OVN Masquerade` would help understanding the comment.

Reply:
@jcaamano can I resolve this comment now ? AFAIU, the comment no longer exists.

Reply:
The comment is still there
```
// Add a neighbour entry on the K8s node to map dummy IPs with MACs. This is
// required because the dummy next hop does not exist on the network and will not
// respond to an ARP, so to route we need an entry. Additionally, the OVN dummy IP
// is not assigned to its interface, so we also need a fake entry for that
```
could it be more easily understood if we use the same terms as in the docs?
```
// Add a neighbour entry on the K8s node to map dummy next-hop masquerade addresses with MACs. This is
// required because these addresses do not exist on the network and will not
// respond to an ARP, so to route we need an entry. Additionally, the OVN Masquerade IP
// is not assigned to its interface, so we also need a fake entry for that.
```